resources/js/bar-chart.js

```diff
@@ -3,23 +3,26 @@ import { Bar } from 'vue-chartjs'
 export default {
   extends: Bar,
   props: {
-    chartData: 'chartData',
-    options: 'options',
+    chartData: {
+      type: Object,
+      required: true
+    },
+    options:{
+      type: Object,
+      required: true
+    },
     height: {
-      default: '120',
+      default: 120,
       type: Number
     }
   },
-  data: () => ({
-    options: {}
-  }),
   methods: {
     renderStackedChart () {
       this.renderChart(this.chartData, this.options)
     }
   },
   mounted () {
-    this.renderChart(this.chartdata, this.options)
+    this.renderChart(this.chartData, this.options)
```

Comment:
I think renderChart should only be called on mounted when Object.keys(this.chartData).length > 0